python/idsse_common/idsse/common/sci/geo_image.py

```diff
@@ -100,6 +114,28 @@ def __init__(self, proj: GridProj, rgb_array: np.ndarray, scale: int = 1):
         self.rgb_array = rgb_array
         self.scale = scale
 
+    @classmethod
+    def from_proj(
+        cls,
+        proj: GridProj,
+        fill_color: Color = (255, 255, 255),
+        scale: int = 1
+    ) -> Self:
+        """Method for building a geographical image without background data.
+
+        Args:
+            proj (GridProj): Grid projection to be used for this geo image, must match data_grid.
+            fill_color (Color): The color to fill the image with. Default is white.
+            scale (int, optional): The height and width that a grid cell will be scaled to in the
+                                   image. Defaults to 1.
+
+        Returns:
+            Self: GeoImage
+        """
+        rgb_array = np.zeros((proj.width*scale, proj.height*scale, 3), np.uint8)
+        rgb_array[...] = fill_color
```

Comment:
I've never seen this syntax in Python, what does this do? Is it shorthand for looping through all the values in the array and setting them to `fill_color`?